metaseq/checkpoint_utils.py

```diff
@@ -209,6 +210,7 @@ def load_checkpoint(cfg: CheckpointConfig, trainer, **passthrough_args):
                             if not f.startswith("_")
                         ]
                     )
+                    assert len(present_files) > 0
```

Comment:
Nit: `present_files` is an int here already (think you might have some uncommitted changes to add)